Bottom: f7b6651f7950a2356525cce7d8d238bcb4ecb209
Top:    8689918b360f1ce447ae13b148ffad8bedc17eb5
Author: Sergey Alyoshin <alyoshin.s@gmail.com>
Date:   2010-05-26 23:21:28 +0100

Add top level menu to localization


---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 37e141d..d0355d6 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -78,6 +78,10 @@ MainMenu =
 #
 # Edit Menu
 #
+<<<<<<< current
+=======
+
+>>>>>>> patched
   {"Edit"
    {"Undo last operation" Undo() a={"U" "<Key>u"}}
    {"Redo last undone operation" Redo() a={"Shift-R" "Shift<Key>r"}}
@@ -116,6 +120,10 @@ MainMenu =
 #
 # View Menu
 #
+<<<<<<< current
+=======
+
+>>>>>>> patched
   {"View"
    {"Enable visible grid" checked=drawgrid Display(Grid)}
    {"Grid units"
